torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1809,39 +1809,46 @@ def sample_inputs_floor_divide(op_info, device, dtype, requires_grad):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
 
-        SampleInput(make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad),
-                    args=(bernoulli_scalar().to(device),
-                          make_tensor((S, S), device, dtype, low=None, high=None, requires_grad=requires_grad))),
-    )
+    def samples_generator():
+        yield SampleInput(make_arg((S, S)), args=(torch.randn(S, S, device=device) > 0, make_arg((S, S))))
+        yield SampleInput(make_arg((S, S)), args=(torch.randn((S,), device=device) > 0, make_arg((S, S))))
+        yield SampleInput(make_arg((S, S)), args=(bernoulli_scalar().to(device), make_arg((S, S))))
+
+        if not for_inplace_variant:
+            yield SampleInput(make_arg((S,)), args=(torch.randn(S, S, device=device) > 0, make_arg((S, S))))
 
+    samples = tuple(samples_generator())
     return samples
 
-def sample_inputs_masked_fill(op_info, device, dtype, requires_grad):
-    def _make_tensor_helper(shape):
-        return make_tensor(shape, device, dtype, low=None, high=None, requires_grad=requires_grad)
 
-    samples = (
-        SampleInput(_make_tensor_helper((S, S)), args=(torch.randn(S, S, device=device) > 0, 10)),
-        SampleInput(_make_tensor_helper((S, S)), args=(torch.randn(S, S, device=device) > 0, _make_tensor_helper(()))),
-        SampleInput(_make_tensor_helper((S, S)), args=(torch.randn(S, device=device) > 0, 10)),
-        SampleInput(_make_tensor_helper(()), args=(torch.randn((), device=device) > 0, 10)),
-        SampleInput(_make_tensor_helper(()), args=(torch.randn((), device=device) > 0, _make_tensor_helper(()))),
-        SampleInput(_make_tensor_helper((S, S)), args=(torch.randn((), device=device) > 0, 10)),
-    )
+def sample_inputs_masked_fill(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
 
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
+
+    def sample_generator():
+        yield SampleInput(make_arg((S, S)), args=(torch.randn(S, S, device=device) > 0, 10))
+        yield SampleInput(make_arg((S, S)), args=(torch.randn(S, S, device=device) > 0, make_arg(())))
+        yield SampleInput(make_arg((S, S)), args=(torch.randn(S, device=device) > 0, 10))
+        yield SampleInput(make_arg(()), args=(torch.randn((), device=device) > 0, 10))
+        yield SampleInput(make_arg(()), args=(torch.randn((), device=device) > 0, make_arg(())))
+        yield SampleInput(make_arg((S, S)), args=(torch.randn((), device=device) > 0, 10))
+
+        if not for_inplace_variant:
+            yield SampleInput(make_arg((S,)),
+                              args=(torch.randn(S, S, device=device) > 0, make_arg(())))
+            yield SampleInput(make_arg((S,)),
+                              args=(torch.randn(S, S, device=device) > 0, 10))
```

Comment:
Why this change instead of using the bernoulli op?

Reply:
`bernoulli` produces scalar tensor however we want (S, S) tensor.
https://github.com/pytorch/pytorch/blob/3d492b069767dd814fdf72129074fc0f4511f400/torch/testing/_internal/common_methods_invocations.py#L3908-L3909

Reply:
```
('masked_fill', (M,), (torch.BoolTensor(M, M).bernoulli_(), 10), 'broadcast_lhs'),
```

I guess I mean where did this case go? We don't have to worry about it in this PR.

Reply:
Ah, replaced it with ( semantically both are doing the same thing),
```python
SampleInput(make_arg((S,)),
            args=(torch.randn(S, S, device=device) > 0, 10)
```

Reply:
OK